app/models/file_depot.rb

```diff
@@ -16,6 +16,11 @@ def self.supported_protocols
     @supported_protocols ||= subclasses.each_with_object({}) { |klass, hash| hash[klass.uri_prefix] = klass.name }.freeze
   end
 
+  def self.depot_description_to_class(description)
+    class_name = supported_depots.key(description)
```

Comment:
According to http://brakemanscanner.org/blog/2015/09/23/brakeman-3-dot-1-1-released, Brakeman was updated to `Treat a.try(&:b) like a.b()`